test/functional/test_framework/test_node.py

```diff
@@ -140,6 +150,17 @@ def __init__(self, i, datadir, *, chain, rpchost, timewait, bitcoind, bitcoin_cl
             AddressKeyPair('mzRe8QZMfGi58KyWCse2exxEFry2sfF2Y7', 'cPiRWE8KMjTRxH1MWkPerhfoHFn5iHPWVK5aPqjW8NxmdwenFinJ'),
     ]
 
+    def get_valgrind_version(self):
+        """Returns valgrind version (major, minor) in a tuple or None if not found"""
```

Comment:
nit: s/Returns/Return/ like the other docstrings

Reply:
Ok :)